src/components/cards/TappableCard.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+import * as React from 'react'
+import { TouchableOpacity, View } from 'react-native'
+import FontAwesome5 from 'react-native-vector-icons/FontAwesome5'
+
+import { memo } from '../../types/reactHooks.js'
+import { type Theme, cacheStyles, useTheme } from '../services/ThemeContext.js'
+import { Card } from './Card'
+
+/**
+ * An (optionally) tappable card that displays its children in up to two left/right
+ * sections. If the card is configured to be tappable, a chevron is drawn on the
+ * right side of the card.
+ */
+const TappableCardComponent = ({
+  leftChildren,
+  nonTappable = false,
+  onPress,
+  rightChildren
+}: {
+  leftChildren: React.Node,
+  nonTappable?: boolean,
+  onPress?: any => void | (any => Promise<void>),
+  rightChildren?: React.Node
+}) => {
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity onPress={onPress}>
+        <Card>
+          <View style={styles.cardContainer}>
+            <View style={styles.spacedContainer}>
+              <View style={styles.leftContainer}>{leftChildren}</View>
+              <View style={styles.rightContainer}>{rightChildren}</View>
```

Comment:
Why not just `children` instead of `leftChildren` and `rightChildren` being separate? The parent component can pass any number of children it can get away with fitting within the card.